internal/venice-client-common/src/main/java/com/linkedin/venice/schema/writecompute/WriteComputeSchemaConverter.java

```diff
@@ -446,26 +443,4 @@ public Schema getNoOpOperation(String namespace) {
     noOpSchema.setFields(Collections.emptyList());
     return noOpSchema;
   }
-
-  public static WriteComputeOperation getFieldOperationType(Object writeComputeFieldValue) {
-    Utils.notNull(writeComputeFieldValue);
-
-    if (writeComputeFieldValue instanceof IndexedRecord) {
-      IndexedRecord writeComputeFieldRecord = (IndexedRecord) writeComputeFieldValue;
-      String writeComputeFieldSchemaName = writeComputeFieldRecord.getSchema().getName();
-
-      if (writeComputeFieldSchemaName.equals(NO_OP_ON_FIELD.name)) {
-        return NO_OP_ON_FIELD;
-      }
-
-      if (writeComputeFieldSchemaName.endsWith(LIST_OPS.name)) {
-        return LIST_OPS;
-      }
-
-      if (writeComputeFieldSchemaName.endsWith(MAP_OPS.name)) {
-        return MAP_OPS;
-      }
```

Comment:
I see your concern, but I think it is more than just namespace right? 
For example, here is one field schema I print out that get passed into the util method:

> {"type":"record","name":"nullableMapFieldMapOps","namespace":"com.linkedin.avro","fields":[{"name":"mapUnion","type":{"type":"map","values":"int"},"default":{}},{"name":"mapDiff","type":{"type":"array","items":"string"},"default":[]}]}

For this one, if you when to instanceOf MapOps, then I think even the internal operation types should match right? But for different map type, it can be different?

Reply:
Yeah, the type of the item in the collection can change, which makes this tricky... I wonder if there's a better way 🤔 ...